From 76edb04c04f925b827f0ea7b05b056b95fa4ddac Mon Sep 17 00:00:00 2001
From: Patrick Hunziker <patrick@hunziker.io>
Date: Sun, 8 Jun 2025 10:06:08 +0200
Subject: [PATCH 20/21] dts: Add bcm2712-rpi-5-b.dtb to DTB list

Registers the bcm2712-rpi-5-b.dtb for Raspberry Pi 5 support in the DTS Makefile.

Signed-off-by: appkins <nbatkins@gmail.com>
---
 arch/arm/dts/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 267b0179a5f..bef426365a0 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -1028,7 +1028,8 @@ dtb-$(CONFIG_ARCH_BCM283X) += \
 	bcm2837-rpi-3-b.dtb \
 	bcm2837-rpi-3-b-plus.dtb \
 	bcm2837-rpi-cm3-io3.dtb \
-	bcm2711-rpi-4-b.dtb
+	bcm2711-rpi-4-b.dtb \
+  bcm2712-rpi-5-b.dtb

 dtb-$(CONFIG_TARGET_BCMNS) += ns-board.dtb

--
2.52.0

